src/libraries/System.Private.CoreLib/src/System/Globalization/CultureInfo.cs

```diff
@@ -462,7 +467,12 @@ public static CultureInfo InvariantCulture
         {
             get
             {
+// https://github.com/dotnet/runtime/issues/94225
+#if TARGET_BROWSER
+                s_InvariantCultureInfo ??= new CultureInfo(CultureData.Invariant, isReadOnly: true);
+#else
                 Debug.Assert(s_InvariantCultureInfo != null);
```

Comment:
Are we sure that `s_InvariantCultureInfo != null` when `#if TARGET_BROWSER`? I think we could keep the assert even for wasm.